deeppavlov/configs/go_bot/gobot_dstc2_best.json

```diff
@@ -119,8 +119,8 @@
       "{DEEPPAVLOV_PATH}/requirements/en_core_web_sm.txt"
     ],
     "labels": {
-      "telegram_utils": "GoalOrientedBot",
-      "server_utils": "GoalOrientedBot"
+      "telegram": "GoalOrientedBot",
```

Comment:
I think that @seliverstov meant to change the directory names and not configs attributes

Reply:
Definitely. @varenick, you shouldn't change the number of parameters.